pages/linux/isoinfo.md

```diff
@@ -0,0 +1,15 @@
+# isoinfo
+
+> Utility programs for dumping and verifying iso images.
+
+- List all the files packed into an iso:
+
+`isoinfo -f -i {{path/to/image.iso}}`
+
+- Extract a specific file to stdout:
+
+`isoinfo -i {{path/to/image.iso}} -x {{/PATH/TO/FILE/INSIDE/ISO.EXT}}`
```

Comment:
It's part of the ISO disk image standard I think. All the ones I inspected also had `;1` appended to them too, which was essential when using this command I found.